mne/io/ctf/ctf.py

```diff
@@ -217,11 +217,11 @@ def _get_sample_info(fname, res4, system_clock):
     else:
         n_trial = n_samp // res4['nsamp']
         n_omit = n_samp_tot - n_samp
-        n_samp = n_trial * res4['nsamp']
```

Comment:
That line should probably change to just use `n_samp` now, right? It seems like `n_samp_tot` isn't actually the size on disk, unless the zeros actually get written to disk.